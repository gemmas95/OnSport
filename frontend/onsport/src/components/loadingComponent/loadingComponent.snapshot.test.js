import renderer from 'react-test-renderer';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import LoadingComponent from './loadingComponent';

describe('LoadingComponent', () => {
    const loadingComponentTree = renderer.create(
        <Router>
            <LoadingComponent />
        </Router>
    );
    it('should match', () => {
        expect(loadingComponentTree).toMatchSnapshot();
    });
});
