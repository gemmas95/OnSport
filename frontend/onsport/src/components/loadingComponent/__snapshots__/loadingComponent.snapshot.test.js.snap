// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingComponent should match 1`] = `
<div
  className="mainContainer__centered mainContainer"
>
  <img
    alt="loading_gif"
    src="https://i.pinimg.com/originals/cd/23/fb/cd23fb22066170c0e25ed35f2a487ce2.gif"
  />
  <p>
    Loading...
  </p>
</div>
`;
